### PR TITLE
Feature/1379 Update e2e tests for sprint 31

### PR DESCRIPTION
**Pull request by andrew-bowers-tcg**
Updated e2e tests. Added object paradigm to pass common form data around

### CODE REVIEW

**Review by dheitzer — CHANGES_REQUESTED** [review]
We may want to clean up some of the "code smells' that sonarqube found - many look low-hanging like unused code and imports

image

File: front-end/cypress/e2e/pages/contactListPage.ts [hunk 1]
```diff
@@ -63,6 +17,7 @@ export class ContactListPage {
 
     if (formData['contact_type'] == 'Individual' || formData['contact_type'] == 'Candidate') {
       //Contact
+      console.log(formData);
```
Comment (dheitzer) [inline front-end/cypress/e2e/pages/contactListPage.ts]: might want to remove this

File: front-end/cypress/e2e/pages/transactionDetailPage.ts [hunk 2]
```diff
@@ -1,79 +1,158 @@
...
+    }
+    if (formData.election_other_description) {
+      cy.get(alias).find('#election_other_description').safeType(formData.election_other_description);
+    }
+    if (formData.purpose_description) {
+      cy.get(alias).find('textarea#purpose_description').safeType(formData.purpose_description);
+    }
+    cy.get(alias).find('textarea#text4000').safeType(formData.memo_text);
+    
+    if (formData.category_code != "") {
```
Comment (dheitzer) [inline front-end/cypress/e2e/pages/transactionDetailPage.ts]: In this and other places - might want to use !== instead to avoid unexpected behavior

**Review by dheitzer — DISMISSED** [review]
Just a couple minor items - otherwise looks good!

File: front-end/cypress/e2e/pages/transactionDetailPage.ts (hunk 2 above)
Comment (mjtravers) [reply front-end/cypress/e2e/pages/transactionDetailPage.ts]: made this update

File: front-end/cypress/e2e/pages/contactListPage.ts (hunk 1 above)
Comment (mjtravers) [reply front-end/cypress/e2e/pages/contactListPage.ts]: console.log has been removed

**Review by mjtravers — APPROVED** [review]

**Pull request merged**